bigflow/cli.py

```diff
@@ -188,6 +189,7 @@ def find_root_package(project_name: Optional[str], project_dir: Optional[str]) -
     @return: Path
     """
     if project_name is not None:
+        project_name = project_name.replace("-", "_")
```

Comment:
Why do we change this in other way here https://github.com/allegro/bigflow/blob/0e0415bdd6035753e4a7eb1f067fd15df894ae52/bigflow/cli.py#L192

Reply:
Python uses "-" in package names, but "_" in module / file names.